frontend/src/metabase/dashboard/containers/CreateDashboardForm.tsx

```diff
@@ -104,6 +103,7 @@ function CreateDashboardForm({
   return (
     <FormProvider
       initialValues={computedInitialValues}
+      enableReinitialize
```

Comment:
why do we need this?

Reply:
that makes sense, I think some of our UIs kind of assume certain conditions e.g. our navbar loads quite a lot of things already. But our SDK components should be treated as standalone components.

Reply:
But just to be sure try to see if this flag loads any additional requests or not, ideally, if it uses the cache then the behavior should stay the same. But if they fire a new request, a new render might cause some difference in the behavior.